build/.goreleaser.yml

```diff
@@ -76,13 +100,66 @@ nfpms:
           - newrelic-infra (>= 1.48.0)
           - nrjmx (>= 2.6.0)
       rpm:
-        file_name_template: "{{ .ProjectName }}-{{ .Version }}-1.{{ .Arch }}"
-        replacements:
-          amd64: x86_64
         dependencies:
           - newrelic-infra (>= 1.48.0)
           - nrjmx >= 2.6.0
+        file_name_template: >-
+          {{- .ProjectName }}-
+          {{- .Version }}-1.
+          {{- if eq .Arch "amd64" -}}x86_64
+          {{- else -}}
+          {{ .Arch }}
+          {{- end }}
+    # Formats to be generated.
+    formats:
+      - deb
+      - rpm
+
+  - id: linux-fips
+    package_name: nri-jmx-fips
+    file_name_template: "{{ .PackageName }}_{{ .Version }}-1_{{ .Arch }}"
+    vendor: "New Relic, Inc."
+    homepage: "https://www.newrelic.com/infrastructure"
+    maintainer: "New Relic Infrastructure Team <infrastructure-eng@newrelic.com>"
+    description: "New Relic Infrastructure jmx Integration extend the core New Relic\nInfrastructure agent's capabilities to allow you to collect metric and\nlive state data from jmx components and it's FIPS compliant."
+    license: "https://newrelic.com/terms (also see LICENSE installed with this package)"
+
+    builds:
+      - nri-nix-fips
+
+    dependencies:
+      - newrelic-infra-fips (>= 1.20.0)
+
+    bindir: "/var/db/newrelic-infra/newrelic-integrations/bin"
+
+    contents:
+      - src: "jmx-config.yml.sample"
+        dst: "/etc/newrelic-infra/integrations.d/jmx-config.yml.sample"
+      - src: "jvm-metrics.yml.sample"
+        dst: "/etc/newrelic-infra/integrations.d/jvm-metrics.yml.sample"
+      - src: "tomcat-metrics.yml.sample"
+        dst: "/etc/newrelic-infra/integrations.d/tomcat-metrics.yml.sample"
+      - src: "CHANGELOG.md"
+        dst: "/usr/share/doc/nri-jmx/CHANGELOG.md"
+      - src: "README.md"
+        dst: "/usr/share/doc/nri-jmx/README.md"
+      - src: "LICENSE"
+        dst: "/usr/share/doc/nri-jmx/LICENSE"
+      - src: "legacy/jmx-definition.yml"
+        dst: "/var/db/newrelic-infra/newrelic-integrations/jmx-definition.yml"
 
+    overrides:
+      rpm:
+        dependencies:
+          - newrelic-infra (>= 1.48.0)
```

Comment:
This should be like this, not? 1.60 is the minimum for newrelic-infra-fips, here is overriding to newrelic-infra, and also doesn't have the deb override.

```
    overrides:
      deb:
        dependencies:
          - newrelic-infra-fips (>= 1.60.0)
          - nrjmx (>= 2.6.0)
      rpm:
        dependencies:
          - newrelic-infra-fips (>= 1.60.0)
          - nrjmx >= 2.6.0
      ...
```